acceptance/lib/helper.rb

```diff
@@ -193,6 +193,7 @@ def puppet_apply_as_puppet_user
     # installed.
     manifest_path = master.tmpfile("puppetserver_manifest.pp")
     herp_path = master.tmpfile("herp")
+    bin_dir = "/opt/puppetlabs/agent/bin/"
```

Comment:
I think those values are only available from beaker if `type => aio` is set. Not sure that we want to do that because I don't know what other issues that might cause?